src/components/AddTask/AddTask.tsx

```diff
@@ -1,11 +1,169 @@
-import React from 'react';
+import React, { useEffect } from 'react';
+import { useParams, useLocation } from 'react-router-dom';
+import { useMutation } from '@apollo/react-hooks';
+import useForm from 'react-hook-form';
+import { QueryNewbieArgs, Mutation, TaskInput } from 'buddy-app-schema';
+import {
+  makeStyles,
+  Typography,
+  Box,
+  CircularProgress,
+  TextField,
+} from '@material-ui/core';
+import xss from 'dompurify';
+import { ADD_BUDDY_TASK, ADD_NEWBIE_TASK } from 'graphql/add-task.graphql';
+import { ROUTES } from 'shared/routes';
+import withSnackBar from 'decorators/withSnackBar';
+import NavBar from '../NavBar';
+import BackgroundShape from '../BackgroundShape';
+import AppWrapper from '../AppWrapper';
+import RoundedButton from '../RoundedButton';
+import { AddTaskProps } from './types';
+import DICTIONARY from './addTask.dictionary';
+
+const useStyles = makeStyles(theme => ({
+  wrapper: {
+    display: 'flex',
+    flexDirection: 'column',
+    height: '100%',
+  },
+  header: {
+    display: 'flex',
+    alignItems: 'center',
+    justifyContent: 'space-between',
+    margin: theme.spacing(1, 0, 2),
+  },
+  form: {
+    display: 'flex',
+    flexDirection: 'column',
+    height: '100%',
+  },
+  inputWrapper: {
+    flex: 1,
+  },
+  addButton: {
+    margin: theme.spacing(5, 0, 3),
```

Comment:
Maybe if you increase the `3` to `5` it would look better since won't be so close to the bottom.